### PR TITLE
Added Bucket sort algorithm in c

### PR DESCRIPTION
**Pull request by Thati0103**

### CODE REVIEW

**Comment by Anshuvats** [comment]
can i add bucket sort algorithm

File: Bucket_sort/Bucket_sort.c.c [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+typedef struct bucket{
+		double value;					// structure for maintaining buckets
+		struct bucket *next;
+}b;
+
+b* create_node(double p){
+	b *x;
+	x=(b *)malloc(sizeof(b));		//create a new node for inserting new input into the bucket
+	x->value=p;
```
Comment (jainaman224) [inline Bucket_sort/Bucket_sort.c.c]: Fix indentation. You have used various indentation throughout the file. Use 4 space indentation.

Comment (Thati0103) [reply Bucket_sort/Bucket_sort.c.c]: Okay

Comment (jainaman224) [reply Bucket_sort/Bucket_sort.c.c]: It is still pending and please fix file name as well.

Comment (Thati0103) [reply Bucket_sort/Bucket_sort.c.c]: I have added indentation and changed file name

Comment (jainaman224) [reply Bucket_sort/Bucket_sort.c.c]: There is still multiple indentation across file.

Comment (Thati0103) [reply Bucket_sort/Bucket_sort.c.c]: I have done changes. Sorry for the inconvenience. I was unable to identify

**Pull request merged**